cirq-core/cirq/ops/dense_pauli_string.py

```diff
@@ -52,7 +48,7 @@
     pauli_gates.Z,
 ]
 
-TCls = TypeVar('TCls', bound='BaseDensePauliString')
+T = TypeVar('T', bound='BaseDensePauliString')
```

Comment:
Can we revert the name `T` to original `TCls` to minimize unrelated changes? Both the type variables are identical modulo naming.

Reply:
Sure